### ISSUE TITLE
"?" should be avoided too

### ISSUE THREAD

**Issue by paboum** [nohello-net/site#153]
![image](https://github.com/nohello-net/site/assets/54635274/3e2c7188-1f10-4c1d-9729-e7a3c60f7bf6)

Asking any question should specify the information that the questioner is missing and expecting from the questionee. Simply typing "?" is disrespectful at the very least, but serves no communicational purpose, isn't even funny. It should be avoided just as saying "hello" in business communication.

I'd like whoever to expand this site with a section regarding "?" or create a sister site devoted to that issue. I am open to participate in server/domain costs etc.

**Comment by dudynets** [comment]
I completely agree with @paboum's viewpoint on the use of "?" in communication. It's frustrating when a message lacks context and simply consists of a question mark. Such messages can be quite unproductive and leave a lot of room for misunderstanding. I'm all for the idea of creating a sister site dedicated to addressing this issue.

If this initiative moves forward, I'd be more than willing to also contribute to server/domain costs and assist with development if necessary.

Looking forward to the progress!

**Comment by trey7658** [comment]
This should also expand to using the person's name and then a question mark, for example: (name)?

**Comment by paboum** [comment]
Well, underspecified questions are unproductive in many ways, however plain "?" is an extreme example, a marker of total disregard for question's function, asking it just for the sake of asking. Rectifying it will also have impact on all other cases, this is a "mother of all" underspecified questions. I wouldn't divide the efforts to any subclass at the moment.
